liberty-maven-plugin/src/main/java/net/wasdev/wlp/maven/plugins/server/StopServerMojo.java

```diff
@@ -28,7 +28,7 @@
 public class StopServerMojo extends StartDebugMojoSupport {
 
     /**
-     * Timeout to verify stop successfully
+     * Timeout to verify stop successfully, this parameter is ignored
```

Comment:
@patricktiu We were thinking that we could not remove this safely as it might break scripts. Can you confirm?